examples/hello_world/finance/demo.py

```diff
@@ -0,0 +1,112 @@
+
+print(env.summary)
+for ep in range(max_ep):
+    metrics = None
+    decision_evts = None
+    is_done = False
+    actions = None
+
+    while not is_done:
+        metrics, decision_evts, is_done = env.step(actions)
+
+        ep_start = time.time()
+        actions = []
+        if not is_done:
+            for decision_event in decision_evts:
+                if decision_event.action_type == ActionType.order:
+                    # stock trading decision
+                    stock_index = decision_event.item
+                    action_scope = decision_event.action_scope
+                    last_frame_idx = tick_to_frame_index(start_tick, env.tick-1, snapshot_resolution)
+                    min_amount = action_scope.buy_min
+                    max_amount = action_scope.buy_max
+                    sell_min_amount = action_scope.sell_min
+                    sell_max_amount = action_scope.sell_max
+                    supported_order_types = action_scope.supported_order
+                    print("<<<<    trade decision_event", stock_index, min_amount, max_amount, sell_min_amount, sell_max_amount)
+                    # qurey snapshot for stock information
+                    cur_env_snap = env.snapshot_list['stocks']
+                    holding = cur_env_snap[last_frame_idx:int(decision_event.item):"account_hold_num"][-1]
+                    cost = cur_env_snap[last_frame_idx:int(decision_event.item):"average_cost"][-1]
+                    opening_price = cur_env_snap[last_frame_idx:int(decision_event.item):"opening_price"][-1]
+                    closing_price = cur_env_snap[last_frame_idx:int(decision_event.item):"closing_price"][-1]
+                    highest_price = cur_env_snap[last_frame_idx:int(decision_event.item):"highest_price"][-1]
+                    lowest_price = cur_env_snap[last_frame_idx:int(decision_event.item):"lowest_price"][-1]
+                    adj_closing_price = cur_env_snap[last_frame_idx:int(decision_event.item):"adj_closing_price"][-1]
+                    dividends = cur_env_snap[last_frame_idx:int(decision_event.item):"dividends"][-1]
+                    splits = cur_env_snap[last_frame_idx:int(decision_event.item):"splits"][-1]
+                    #print(holding, cost, opening_price, closing_price, highest_price, lowest_price, adj_closing_price, dividends, splits)
+                    # qurey snapshot for account information
+                    total_money = env.snapshot_list['account'][last_frame_idx:0:"total_money"][-1]
+                    remaining_money = env.snapshot_list['account'][last_frame_idx:0:"remaining_money"][-1]
+                    #print("env.tick: ", env.tick, " holding: ", holding, " cost: ", cost, "total_money:", total_money, "remaining_money", remaining_money)
+
+                    if holding > 0:  # sub_engine_name -> market
+                        # action = MarketOrder(item=decision_event.item, amount=holding,
```

Comment:
I think we need to demo a reasonable simple strategy here, not list all.

Reply:
ok